cockroachcloud/export-logs.md

```diff
@@ -76,7 +73,7 @@ Perform the following steps to enable log export from your {{ site.data.products
 	1. Select **Roles** and click **Create role**.
 	1. For **Trusted entity type**, select **AWS account**.
 	1. Choose **Another AWS account**.
-		1. For **Account ID**, provide the {{ site.data.products.dedicated }} AWS Account ID that you found previously by querying your cluster's Cloud API.
+	1. For **Account ID**, provide the {{ site.data.products.dedicated }} AWS Account ID from step 3.
```

Comment:
nit: possible confusion here about "which step 3" since it's a nested list. but not a huge dela.

Reply:
Ah, In the rendered HTML, this renders as step 4.e. Step 3, the target, is unnested.